app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -368,11 +368,7 @@ private fun RecentlyVisitedMenu(
 private fun RecentlyVisitedDivider(
```

Comment:
Could we remove this wrapper function while we are at it?

Reply:
yes, I'll remove it